crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -32,6 +34,26 @@ impl Settlement {
     /// id.
     const META_DATA_LEN: usize = 8;
 
+    pub fn auction_id(&self) -> auction::Id {
+        self.auction_id
+    }
+
+    pub fn score(
+        &self,
+        prices: &auction::Prices,
+        policies: &HashMap<OrderUid, Vec<fee::Policy>>,
+    ) -> Result<eth::Ether, trade::Error> {
+        self.trades
+            .iter()
+            .map(|trade| {
+                trade.score(
+                    prices,
+                    &policies.get(trade.order_uid()).cloned().unwrap_or_default(),
```

Comment:
The clone could be avoided.
```suggestion
                    policies
                        .get(trade.order_uid())
                        .map(|value| value.as_slice())
                        .unwrap_or_default(),
```

Reply:
You are right! fixed